packages/design-system/src/components/circlePieChart/emptyCirclePieChart.tsx

```diff
@@ -18,7 +18,7 @@
  */
 import React from 'react';
 import { VictoryPie } from 'victory';
-import { COLOR_MAP } from '@ps-analysis-tool/design-system';
+import { COLOR_MAP } from '@ps-analysis-tool/common';
```

Comment:
Could you explain the reason for moving the color map? In my opinion, it is more of a design-system constant.

Reply:
Yes because COLOR_MAP is also used in common package and it being in design system makes a duplicate copy. If I were to reverse it then I would be creating circular dependency for between design-system and common package

Reply:
Let's move `prepareCookieStatsComponents` to design-system then. It can be classified as a design utility IMO.

Reply:
Its being used in popup and cli-dashboard too will need to change there as well. Then doesnt it make sense to keep it in common package?

Reply:
Pop-up and cli-dashboard can also use a design utility that generates theming-related data.

Reply:
Fixed in latest commits.